pom.xml

```
@@ -6,7 +6,7 @@
 
     <groupId>org.carlspring.strongbox</groupId>
     <artifactId>strongbox-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-issue-1877-upgd-dep-SNAPSHOT</version>
```

```suggestion
    <version>1.0-SNAPSHOT</version>
```

Thanks, I'll revert this to the snapshot version we use.